packages/app/src/components/time-series-chart/components/timespan-annotation.tsx

```diff
@@ -58,60 +51,29 @@ export function TimespanAnnotation({
           style={{ mixBlendMode: 'multiply' }}
         />
       );
-    case 'estimate':
+    case 'hatched':
       return (
-        <>
-          <Bar
-            pointerEvents="none"
-            height={height}
-            x={x0}
-            width={width}
-            fill={`url(#${chartId}_estimate_pattern)`}
-          />
-          <Group>
-            <Text
-              fontSize={fontSize}
-              x={x0 - 15}
-              y={15}
-              textAnchor="end"
-              fill="grey"
-            >
-              {config.leftLabel}
-            </Text>
-            <Text
-              fontSize={fontSize}
-              x={x0 + 15}
-              y={15}
-              textAnchor="start"
-              fill="grey"
-            >
-              {config.rightLabel}
-            </Text>
-
-            <Line
-              x1={x0}
-              x2={x0}
-              y1={height}
-              y2={0}
-              stroke="grey"
-              strokeWidth="2"
-            />
-          </Group>
-        </>
+        <Bar
+          pointerEvents="none"
+          height={height}
+          x={x0}
+          width={width}
+          fill={`url(#${chartId}_hatched_pattern)`}
+        />
```

Comment:
I think you should render the hatch pattern over here, extra advantage is that you can get rid of the timespanAnnotation checks in the pattern component, also the id can be defined on this level.
```suggestion
        <>
          <HatchedPattern id={`${chartId}_hatched_pattern`} />
          <Bar
            pointerEvents="none"
            height={height}
            x={x0}
            width={width}
            fill={`url(#${chartId}_hatched_pattern)`}
          />
        </>
```

Reply:
Ok, it also means we can only add one hatched annotation, otherwise we're duplicating the pattern. Thats why it was in the defs section in the first place.

Reply:
I don't think duplicating a pattern is a big issue :-)

Reply:
But I also fail to see why the change is necessary then. I don't think I'm adding an incredible amount of complexity here...

Reply:
Well I actually think it is:
- there's an unclear link between a HatchedPattern and a specific case of a timespanannotation
- It's weird that a pattern should receive information about timespan annotations
- The timespanAnnotation component could be rendered by different chart implementations (e.g. vertical bar chart, mini bar chart). It would be a broken component because it has unclear dependencies on a component rendered somewhere else.

Reply:
Hm, I think I placed the hatch-pattern component inside the 'components' directory of the time-series-chart. WHich kind of implies its tied to that component.
Currently, the hatched pattern is tied to the timespan annotation, that's why I placed the check inside that component, with the idea that if more patterns would be needed in the future this logic would be lifted outside this component to more upstream logic that would decide which patterns wound need to be rendered or not.

Reply:
I understand, but it's still unnecessarily complex if there's a simpler implementation possible?